textproc/stanford-corenlp/Portfile

```diff
@@ -0,0 +1,255 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github  1.0
+PortGroup           java    1.0
+
+name                stanford-corenlp
+version             3.9.2
+revision            0
+github.setup        stanfordnlp CoreNLP ${version} v
+
+categories          textproc java
+platforms           darwin
+license             GPL-3+
+maintainers         nomaintainer
+
+description         A Java suite of core NLP tools
+long_description    Stanford CoreNLP provides a set of natural language \
+                    analysis tools written in Java. It can take raw \
+                    human language text input and give the base forms \
+                    of words, their parts of speech, whether they are \
+                    names of companies, people, etc., normalize and \
+                    interpret dates, times, and numeric quantities, \
+                    mark up the structure of sentences in terms of \
+                    phrases or word dependencies, and indicate which \
+                    noun phrases refer to the same entities. It was \
+                    originally developed for English, but now also \
+                    provides varying levels of support for (Modern \
+                    Standard) Arabic, (mainland) Chinese, French, \
+                    German, and Spanish. Stanford CoreNLP is an \
+                    integrated framework, which makes it very easy to \
+                    apply a bunch of language analysis tools to a \
+                    piece of text. Starting from plain text, you can \
+                    run all the tools with just two lines of code. Its \
+                    analyses provide the foundational building blocks \
+                    for higher-level and domain-specific text \
+                    understanding applications. Stanford CoreNLP is a \
+                    set of stable and well-tested natural language \
+                    processing tools, widely used by various groups in \
+                    academia, industry, and government. The tools \
+                    variously use rule-based, probabilistic machine \
+                    learning, and deep learning components.
+
+homepage            http://stanfordnlp.github.io/CoreNLP/
+
+github.livecheck.regex \
+                    {([0-9.]+)}
+
+use_zip             yes
+github.tarball_from \
+                    archive
+
+checksums           rmd160  97f6bbcc3cf598251c36dbd2fcd8d51b6a6ac85d \
+                    sha256  cf46d0ee23410a5e2b0e43af33a4e7e42a958571f21479b3c0b305a13ce6c14a \
+                    size    74785152
+
+variant stanford_mirror \
```

Comment:
what is the purpose of this variant? Is there ever a need to use another ```master_sites``` than the default MacPort ones; especially since you mention it's slow. To me this seems to add only unnecessary clutter to the Portfile.

Reply:
The Stanford site has a full build and is their “official” repo, albeit quite large and through a soda straw. I thought about deleting the variant, but it’s useful as a reference for the `ant`-built one from the GitHub repo.

Reply:
so this is kind of a “pre-compiled”, ready-to-use version? If so, the variant name doesn’t really reflect what it’s about. But again, I see no reason to provide this, we prefer to build from sources anyway. I do not understand how it’s useful as a reference... for whom? I vote for removal.

Reply:
I’m neither a Java nor an Xcode expert. All I know is that Stanford has a full install with all `.jar`s that’s huge and slow, and that GitHub’s is small(er) and fast because it uses `ant` To build the `.jar`s.

I’m pushing back against removing Stanford’s variant because if anyone is on a platform that fails to build with `ant` they can just install all the `.jar`s from Stanford’s mirror.

Reply:
A agree with @reneeotten that this variant should be removed. No other port does this. Also, if someone wants the officially built version, they should download it themselves directly.

Reply:
Okay, outvoted, done.